### DISCUSSION TITLE
Support for Ruby 3? (release coming shortly)

### DISCUSSION THREAD

**Discussion by jaredcwhite** [vercel/vercel#5534] [General]
The official release of Ruby 3 is coming soon (Christmas I believe) with many exciting new features.

I'd love to see support for that get added to Vercel's Ruby runtime once it's out.

**Comment by styfle** [comment]
We can add support once our [upstream provider](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) adds Ruby 3, thanks!

**Comment by gkunwar** [comment]
@jaredcwhite @styfle no progress yet?
